sw/cpp/error_leds/README.md

```diff
@@ -0,0 +1,17 @@
+CHERI error test for Sonata
+==========================
+
+This triggers each type of CHERIoT exception.
+See Table 7.3 in the [CHERIoT ISA](https://www.microsoft.com/en-us/research/uploads/prod/2023/02/cheriot-63e11a4f1e629.pdf).
+
+Building
+--------
+
+This is built with a simple Makefile that expects to be passed the locations of various tools and the RTOS directory as part of the make invocation:
+
+```
+$ make CHERIOT_LLVM_ROOT=path/to/llvm/bin/ CHERIOT_RTOS_SDK=/path/to/cheriot-rtos/sdk/
+```
```

Comment:
lowRISC/OT [style guide](https://opentitan.org/book/doc/contributing/style_guides/markdown_usage_style.html#code-blocks)

````suggestion
```sh
make CHERIOT_LLVM_ROOT=path/to/llvm/bin/ CHERIOT_RTOS_SDK=/path/to/cheriot-rtos/sdk/
```
````